command.go

```diff
@@ -896,3 +913,63 @@ func (c *commandContext) SelectSegment(p mat.Vec3) {
 	c.UnsetCursors()
 	c.selectMode = selectModeMask
 }
+
+func (c *commandContext) SelectLabelSegment(p mat.Vec3) error {
+	it, err := c.editor.pp.Vec3Iterator()
+	if err != nil {
+		return err
+	}
+	lt, err := c.editor.pp.Uint32Iterator("label")
+	if err != nil {
+		return err
+	}
+
+	res := float32(c.segmentationDistance)
+	w := int(c.segmentationRange / c.segmentationDistance)
+	half := float32(w) * res / 2
+	v := vgs.New(res, [3]int{w, w, w}, p.Sub(mat.Vec3{half, half, half}))
+
+	n := c.editor.pp.Points
+	for i := 0; i < n; i++ {
+		c.selectMask[i] &= ^uint32(selectBitmaskSegmentSelected)
+		a, ok := v.Addr(it.Vec3())
+		if ok {
+			if c.selectMask[i]&(selectBitmaskCropped|selectBitmaskOnScreen) == selectBitmaskOnScreen {
+				v.AddByAddr(a, i)
+			}
+		}
+		it.Incr()
+	}
+	if it, err = c.editor.pp.Vec3Iterator(); err != nil {
+		return err
+	}
+
+	vIndice := v.Storage().Indice()
```

Comment:
It's not needed to be stored to VoxelGrid storage.
```suggestion
	vIndice := make([]int, 0, ??)
	n := c.editor.pp.Points
	for i := 0; i < n; i++ {
		c.selectMask[i] &= ^uint32(selectBitmaskSegmentSelected)
		if c.selectMask[i]&(selectBitmaskCropped|selectBitmaskOnScreen) == selectBitmaskOnScreen {
			continue
		}
		v := it.Vec3At(i).Sub(p)
		if v[0] < -c.segmentationRange || c.segmentationRange < v[0] ||
			v[1] < -c.segmentationRange || c.segmentationRange < v[1] ||
			v[2] < -c.segmentationRange || c.segmentationRange < v[2] {
			continue
		}
		vIndice = append(vIndice, i)
	}
```

Reply:
I made some adjustments to your suggestion in 376d67eb4165aba9cad35d561fcf906649b6845f.